pkg/arvo/lib/hood/drum.hoon

```diff
@@ -237,10 +237,11 @@
     =>  (se-born | %home %group-push-hook)
     (se-born | %home %group-pull-hook)
   =?  ..on-load  (lte hood-version %9)
-   (se-born | %home %graph-store)
+    (se-born | %home %graph-store)
   =?  ..on-load  (lte hood-version %10)
     =>  (se-born | %home %graph-push-hook)
-    (se-born | %home %graph-pull-hook)
+    =>  (se-born | %home %graph-pull-hook)
+    (se-born | %home %herm)
```

Comment:
Not sure what's up with the proliferation of versions here. The latest of these gets run on every `+on-load` call to hood, and remain silent even if the app has already been started.